src/components/explorer/ExplorerIndividualContent.js

```diff
@@ -55,6 +57,12 @@ const ExplorerIndividualContent = () => {
         }
     }, [location]);
 
+    useEffect(() => {
+        if (individualUrl) {
+            dispatch(setIndividualExplorerUrl(individualUrl));
```

Comment:
this feels like a bit of an anti-pattern - is there no part of the route match that works instead? or does that end up matching nested routes instead, so this is needed? if the latter is the case maybe a comment explaining why this way was chosen would be good

Reply:
It was a poor attempt to reduce prop drilling in nested routes. Changed the store to save the current individualId instead.